miner/block_builder.go

```diff
@@ -202,11 +202,16 @@ func (t *BlockBuilder) createBlock(id types.LayerID, atxID types.AtxId, eligibil
 		return nil, errors.New("cannot create blockBytes in genesis layer")
 	} else if id == config.Genesis+1 {
 		votes = append(votes, config.GenesisId)
-	} else {
+	} else { // get from hare
 		bottom, top := calcHdistRange(id, t.hdist)
 		votes, err = t.hareResult.GetResult(bottom, top)
 		if err != nil {
-			return nil, errors.New(fmt.Sprintf("didn't receive hare results for layers bottom=%v top=%v hdist=%v err=%v", bottom, top, t.hdist, err))
+			t.With().Error("GetResult returned an error",
```

Comment:
bad log print, `GetResult` doesn't say much on the context, replace with something more meaningful

Reply:
Done